internal/command/launch/launch.go

```diff
@@ -437,9 +432,9 @@ func shouldAppUseMachinesPlatform(ctx context.Context, orgSlug string) (bool, er
 	return orgDefault, nil
 }
 
-func appExists(ctx context.Context, cfg *appconfig.Config) (bool, *api.AppBasic, error) {
+func appExists(ctx context.Context, cfg *appconfig.Config) (bool, *api.App, error) {
 	client := client.FromContext(ctx).API()
-	app, err := client.GetAppBasic(ctx, cfg.AppName)
+	app, err := client.GetApp(ctx, cfg.AppName)
```

Comment:
oh no, this call is way more expensive. if we have to do this then better to revert and use the previous attempt

Reply:
ok i've reverted the commit.